draft-birkholz-rats-corim.md

```diff
@@ -789,10 +793,629 @@ Not included in the full CDDL definition are CDDL dependencies to CoSWID. The fo
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoMID Triples Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | comid.reference-triples | RFC-AAAA
+| 1 | comid.endorsed-triples | RFC-AAAA
+| 2 | comid.identity-triples | RFC-AAAA
+| 3 | comid.attest-key-triples | RFC-AAAA
+| 4-255 | Unassigned
+{: #tbl-iana-comid-triples-map-items title="CoMID Triples Map Items Initial Registrations"}
+
+{: #iana-comid-environment-map-items}
+## CoMID Environment-Map Items Registry
+
+This document defines a new registry titled "CoMID Environment Map".
+The registry uses integer values as index values for items in 'comid-environment-map' CBOR maps.
+
+Future registrations for this registry are to be made based on {{RFC8126}} as follows:
+
+| Range             | Registration Procedures
+|---
+| 0-127    | Standards Action
+| 128-255  | Specification Required
+{: #tbl-iana-comid-environment-map-items-reg-procedures title="CoMID Environment Map Items Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoMID Environment Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | comid.class | RFC-AAAA
+| 1 | comid.instance | RFC-AAAA
+| 2 | comid.group | RFC-AAAA
+| 3-255 | Unassigned
+{: #tbl-iana-comid-environment-map-items title="CoMID Environment Map Items Initial Registrations"}
+
+{: #iana-comid-class-map-items}
+## CoMID Class-Map Items Registry
+
+This document defines a new registry titled "CoMID Class Map".
+The registry uses integer values as index values for items in 'comid-class-map' CBOR maps.
+
+Future registrations for this registry are to be made based on {{RFC8126}} as follows:
+
+| Range             | Registration Procedures
+|---
+| 0-127    | Standards Action
+| 128-255  | Specification Required
+{: #tbl-iana-comid-class-map-items-reg-procedures title="CoMID Class Map Items Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoMID Class Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | comid.class-id | RFC-AAAA
+| 1 | comid.vendor | RFC-AAAA
+| 2 | comid.model | RFC-AAAA
+| 3 | comid.layer | RFC-AAAA
+| 4 | comid.index | RFC-AAAA
+| 5-255 | Unassigned
+{: #tbl-iana-comid-class-map-items title="CoMID Class Map Items Initial Registrations"}
+
+{: #iana-comid-measurement-map-items}
+## CoMID Measurement-Map Items Registry
+
+This document defines a new registry titled "CoMID Measurement Map".
```

Comment:
comid-measurement-map is not currently extensible. As the 'object' of a triple, the predicate semantics can be broken by extensibility. The measurement-values-map is extensible which is likely the right way to extend the map without breaking predicate semantics.

Reply:
flagged for pruning